app/screens/about/learn_more.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Text, TouchableOpacity, View} from 'react-native';
+
+import Config from '@assets/config.json';
+import FormattedText from '@components/formatted_text';
+import {useTheme} from '@context/theme';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+
+import type SystemModel from '@typings/database/models/servers/system';
+
+type LearnMoreProps = {
+    config: SystemModel;
+    onHandleAboutEnterprise: () => void;
+    onHandleAboutTeam: () => void;
+};
+
+const LearnMore = ({config, onHandleAboutEnterprise, onHandleAboutTeam}: LearnMoreProps) => {
+    const theme = useTheme();
+    const style = getStyleSheet(theme);
+
+    let id;
+    let defaultMessage;
+    let onPress;
+    let url;
+
+    id = 'about.teamEditionLearn';
+    defaultMessage = 'Join the Mattermost community at ';
+    onPress = onHandleAboutTeam;
+    url = Config.TeamEditionLearnURL;
```

Comment:
```suggestion
    let id = t('about.teamEditionLearn');
    let defaultMessage = 'Join the Mattermost community at ';
    let onPress = onHandleAboutTeam;
    let url = Config.TeamEditionLearnURL;
```

import `t` from `@i18n`